modules/custom-tuning-example.adoc

```diff
@@ -31,10 +31,12 @@ spec:
     name: openshift-ingress
   recommend:
   - match:
-    - label: tuned.openshift.io/ingress-pod-label
-      value: "ingress-pod-label-value"
-      type: pod
+    - label: tuned.openshift.io/ingress-node-label
     priority: 10
     profile: openshift-ingress
 _EOF_
 ----
+
+Note that custom profile writers are strongly encouraged to include the default tuned daemon profiles
+shipped within the default Tuned CR.  The example above uses the default `openshift-control-plane`
+profile to do just that.
```

Comment:
s/to do just that/to accomplish this.

Reply:
Addressed in https://github.com/openshift/openshift-docs/pull/21325